spec/filters/document_provider_spec.rb

```diff
@@ -277,7 +277,7 @@
         order_by_date: '2017-03-16'
       )
     end
-    let(:tag) { ['test'] }
+    let(:tag) { ['test', 'another-test'] }
```

Comment:
Can we test the filtering with multiple tags in isolation? This context here is more about the ordering, so it tries to use a combination of filters (and I've actually added the tag here – only one 😞– for completeness sake...), but the main goal is to test the ordering indeed.

Something like "when filtering by tags" maybe? And/or maybe extending https://github.com/moneyadviceservice/cms/blob/e987fc050f22d6e733018e3cf2860ae6d36760a2/spec/controllers/api/documents_controller_spec.rb#L112-L141 as well?

Reply:
Yes, I agree. Might not be clear why two tags have been added to the factory and in the future the context (or lack thereof 😄  ) might be lost 👍